content/docs/wrp/simple-messages.md

```diff
@@ -47,9 +54,10 @@ the same message type.
 Name | Description
 -----|--------------
 msg_type | The message type for the request-response.  (**SHALL** be 3.)
-source | The device_id name of the device originating the request or response.
-dest | The device_id name of the target device of the request or response.
-content_type | (optional) The media type of the payload.
+source | The device_id name of the device originating the request or response (i.e. dns:tr1d1um.example.net for inbound messages).
```

Comment:
I wonder if parodus is or plans to use the source field (`dns:tr1d1um.example.net`) on inbound messages.

Reply:
I thought parodus uses the source at the very least to fill in the destination for the response wrp it sends.  I could be wrong.